Makefile

```
@@ -47,7 +47,7 @@ show-coverage: test
 
 setup:
 	$(GOMOD) download
-	$(GOMOD) tidy -compat=1.17
+	$(GOMOD) tidy -compat=1.22.1
```

```suggestion
	$(GOMOD) tidy
```

Nit: could we just drop this?
I highly suspect modern `go mod tidy` uses the `go.mod` to tody,